### PR TITLE
[DELIVERABLE 2] SRS Document: Modified Functional Requirements

### LINKED ISSUE #64: [DELIVERABLE 2] SRS Document: Functional Requirements

### PR DESCRIPTION
**Pull request by nivethakuruparan**
## Description

Added rationale, changed formatting and revised the functional requirements. Should close #64.

## How Has This Been Tested?

Compiled locally

### CODE REVIEW

**Review by Sevhena — APPROVED** [review]
LGTM

File: docs/SRS/SRS.tex [hunk 1]
```diff
@@ -453,52 +453,46 @@ \subsection{Individual Product Use Cases (PUC's)}
...
-  \end{itemize}
+\begin{enumerate}[label=FR \arabic*., wide=0pt, leftmargin=*]
+    \item \emph{The system must accept Python source code files.}\\
+    {\bf Rationale:} The system needs to process Python code as its primary input to refactor and improve energy efficiency.\\
+    {\bf Fit Criterion:} The system successfully processes valid Python files without errors and provides feedback for invalid files.
+    \item \emph{The system must identify specific code smells that can be targeted for energy saving.}\\
+    {\bf Rationale:} Energy inefficiencies are often related to well-known code smells, so identifying them is the first step in improving efficiency.\\
+    {\bf Fit Criterion:} The tool should detect and report at least 80\% of the following code smells: Large Class (LC), Long Parameter List (LPL), Long Method (LM), Long Message Chain (LMC), Long Scope Chaining (LSC), Long Base Class List (LBCL), Useless Exception Handling (UEH), Long Lambda Function (LLF), Complex List Comprehension (CLC), Long Element Chain (LEC), Long Ternary Conditional Expression (LTCE).
```
Comment (Ayushi1972) [inline docs/SRS/SRS.tex:462]: References added?

**Review by Ayushi1972 — APPROVED** [review]
LGTM

File: docs/SRS/SRS.tex (hunk 1 above)
Comment (nivethakuruparan) [reply docs/SRS/SRS.tex:462]: Yeah, it looks like this needs references, I will add them in a different PR

**Pull request merged**